Fundamentals/Code/Subscribing-To-Events/index.md

```diff
@@ -3,6 +3,7 @@ meta.title: "Subscribing to events"
 meta.description: "Subscribing to events allows you to execute custom code on a number of events both before and after the event occurs"
 versionFrom: 8.0.0
 versionTo: 8.18.0
+needsV8Update: "true"
```

Comment:
Would it make sense to link to the "Subscribing the Notifications" here instead? 🤔 
Notification is the equivalent to Events in legacy Umbraco 💪

Reply:
Makes sense. Will link it to subscribing to notifications article.